setup/resource.sh

```diff
@@ -37,10 +37,14 @@ resource_public() {
 }
 
 resource_database() {
+  echo ''
+  echo '» 🚀 Downloading database file... 🚀'
   DATABASE_PATH="$CURRENT_DIR/postgres/entry.d"
 
-  if [ ! -d "$DATABASE_PATH" ]; then
```

Comment:
You don't need to check it exists because we used the .gitkeep file to keep it alive

Reply:
Thanks for your review.